tools/walletextension/main/main.go

```diff
@@ -17,8 +17,10 @@ func main() {
 
 	walletExtensionAddr := fmt.Sprintf("%s:%d", localhost, config.WalletExtensionPort)
 	go walletExtension.Serve(walletExtensionAddr)
-	fmt.Printf("Wallet extension started.\n💡 Visit %s/viewingkeys/ to generate an ephemeral viewing key. "+
-		"Without a viewing key, you will not be able to decrypt the enclave's secure responses to sensitive requests.\n", walletExtensionAddr)
+	fmt.Printf("Wallet extension started.\n💡 Visit http://%s/viewingkeys/ to generate an ephemeral viewing key.\n", walletExtensionAddr)
+	fmt.Println()
+	s, _ := json.MarshalIndent(config, "", "  ")
+	fmt.Printf("Wallet extension config: \n%s", string(s))
```

Comment:
Can we swap these around? Dump the config, then linebreak, then happy message with the `/viewingkeys` link

Reply:
Ya sure 👍